src/javascripts/components/blog/ArticlePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+class ArticlePreview extends React.Component {
+  render() {
+    return (
+      <Link to="/article"><div className="articlePreview">
+        <div className="--imageContainer">
+          <img
+            src="./images/alice-in-wonderland.png"
+            alt="Picture of 'Author'"
+          />
```

Comment:
I saw that you made this image a semi-transparent png, but I've got a challenge for you!

A solid jpg would be a smaller file (and more performant). With a solid jpg (with a white background), then you can try some CSS blend modes to make it look like the design:

https://css-tricks.com/basics-css-blend-modes/